tests/intensive/test_visualization.py

```diff
@@ -277,6 +278,66 @@ def test_similarity_index():
     assert len(results2.points) == len(view)
 
 
+def test_points_field():
+    brain_key = "test_points_field_brain_key"
+    dataset = _load_images_dataset()
+    num_samples = len(dataset)
+    point_field = f"test_point_{rand.randint(0, 1000)}"
+    results = fob.compute_visualization(
+        dataset,
+        brain_key=brain_key,
+        point_field=point_field,
+        seed=51,
+    )
+    sample = dataset.first()
+    example_point = sample.get_values(point_field)
+    assert example_point is not None
+    assert len(example_point) == 2
+    assert isinstance(example_point[0], float)
+    assert len(results.sample_ids) == num_samples
+    assert results.label_ids is None
+    assert np.array_equal(results.points, results.get_points())
+
+    index_name = results.config.point_field_index
+    points = results.get_points()
+    assert len(points) == num_samples
+    assert len(points[0]) == 2
+
+    # cleanup
+    dataset.delete_brain_run(brain_key)
+
+    assert not dataset.has_sample_field(point_field)
+    indexes = dataset.list_indexes()
+    assert index_name not in indexes
+
+
+def test_points_field_patches():
```

Comment:
@brimoor this should cover the intended behavior, but this test is currently failing.